libraries/AP_Math/control.cpp

```diff
@@ -324,6 +324,38 @@ void shape_pos_vel_accel_xy(const Vector2p& pos_input, const Vector2f& vel_input
     shape_vel_accel_xy(vel_target, accel_input, vel, accel, accel_max, jerk_max, dt, limit_total_accel);
 }
 
+// limit_accel_xy limits the acceleration to prioritise acceleration perpendicular to the provided velocity vector.
+bool limit_accel_xy(const Vector2f& vel, Vector2f& accel, float accel_max)
+{
+    // check accel_max is defined
+    if ( !is_positive(accel_max) ) {
```

Comment:
nitpick: we can remove the spaces within the brackets.

Reply:
Done